sdk/test/NetStandard/UnitTests/Core/AWSSDKUtilsTests.cs

```diff
@@ -0,0 +1,29 @@
+using Amazon.Util;
+using Xunit;
+
+namespace UnitTests.NetStandard.Core
+{
+    [Trait("Category", "Core")]
+    public class AWSSDKUtilsTests
+    {
+        [Theory]
+        [InlineData("value, with special chars!", "value%2C%20with%20special%20chars%21")]
+        [InlineData("value, with special chars and path {/+:}", "value%2C%20with%20special%20chars%20and%20path%20%7B%2F%2B%3A%7D")]
+        public void UrlEncodeWithoutPath(string input, string expected)
+        {
+            var encoded = AWSSDKUtils.UrlEncode(input, path: false);
+
+            Assert.Equal(expected, encoded);
+        }
+
+        [Theory]
+        [InlineData("value, with special chars!", "value%2C%20with%20special%20chars!")]
+        [InlineData("value, with special chars and path {/+:}", "value%2C%20with%20special%20chars%20and%20path%20%7B/%2B:%7D")]
```

Comment:
Should I add more data from the valid path chars?

Reply:
You have proven that the characters from the valid path are being taken into account. I don't think this test needs to test each value that is in the collection.

Reply:
I wanted to play extra safe TBH